assets/js/edit.js

```diff
@@ -275,25 +377,105 @@ class Edit extends Component {
 						</PanelRow>
 					</PanelBody>
 				</InspectorControls>
-				<div className={className}>
+				<div className="wp-block-podcasting-podcast-outer">
 					{src ? (
-						<figure key="audio" className={className}>
-							<audio controls="controls" src={src} />
-							{((caption && caption.length) || !!isSelected) && (
-								<RichText
-									tagName="figcaption"
-									placeholder={__(
-										'Write caption…',
-										'simple-podcasting'
+						<>
+							<div className="wp-block-podcasting-podcast__container">
+								{showImage && displayArt && (
+									<div className="wp-block-podcasting-podcast__show-art">
+										<div className="wp-block-podcasting-podcast__image">
+											<img
+												src={showImage}
+												alt={showName}
+											/>
+										</div>
+									</div>
+								)}
+
+								<div className="wp-block-podcasting-podcast__details">
+
+									{displayEpisodeTitle && (
+										<h3 className="wp-block-podcasting-podcast__show-title">
+											{displayEpisodeNumber && (
+												<span>
+													{episodeNumber}.
+												</span>
+											)}
+											{postDetails.title}
+										</h3>
 									)}
-									value={caption}
-									onChange={(value) =>
-										setAttributes({ caption: value })
-									}
-									isSelected={isSelected}
-								/>
-							)}
-						</figure>
+
+									<div className="wp-block-podcasting-podcast__show-details">
+										{displayShowTitle && (
+											<span className="wp-block-podcasting-podcast__title">
+												{showName}
+											</span>
+										)}
+										{displaySeasonNumber && (
+											<span className="wp-block-podcasting-podcast__season">
+												{__(
+													'Season: ',
+													'simple-podcasting'
+												)}
+												{seasonNumber}
+											</span>
+										)}
+										{displayEpisodeNumber && (
+											<span className="wp-block-podcasting-podcast__episode">
+												{__('Episode: ', 'simple-podcasting')}
+												{episodeNumber}
+											</span>
+										)}
+									</div>
+
+									<div className="wp-block-podcasting-podcast__show-details">
+										{displayDuration && (
+											<span className="wp-block-podcasting-podcast__duration">
+												{__('Listen Time: ', 'simple-podcasting')}
+												{duration}
+											</span>
+										)}
+										{displayEpisodeType && (
+											<span className="wp-block-podcasting-podcast__episode-type">
+												{__(
+													'Episode type: ',
+													'simple-podcasting'
+												)}
+												{episodeType}
+											</span>
+										)}
+										{displayExplicitBadge && (
+											<span className="wp-block-podcasting-podcast__explicit-badge">
+												{__(
+													'Explicit: ',
+													'simple-podcasting'
+												)}
+												{explicit}
```

Comment:
I noticed this defaults to a blank value unless the user modifies the dropdown, it needs to default to the attributes default value

Reply:
I'm still seeing this.